src/provisioning/Provisioner.ts

```diff
@@ -862,7 +863,7 @@ export class Provisioner extends ProvisioningApi {
         const roomId = body.matrix_room_id;
         const mappingLogId = `${roomId} <-/-> ${ircDomain}/${ircChannel}`;
 
-        req.log.info(`Provisioning unlink for room ${mappingLogId}`);
+        req.log.info(`Provisioning unlink for room ${mappingLogId} (for ${userId})`);
```

Comment:
Nitpick: for consistency, either remove the parentheses here, or add them to line 789.